src/Components/NotesPage/NotesPage.jsx

```diff
@@ -1,107 +1,91 @@
 import React, { useState, useEffect } from 'react';
-import { useArchive, useAuth, useNotes } from '../../context/';
+import { useModal, useNotes } from '../../context/';
 import { formatDate } from '../../backend/utils/authUtils';
 import { EditorState, convertToRaw, ContentState } from 'draft-js';
-import { MyEditor } from './MyEditor';
+import { Modal } from './modal/Modal';
+import { AddNewNoteCard } from './modal/AddNewNoteCard.jsx';
+import { CreateNewIcon } from '../../assets/svg/allsvg';
+import { NotesCard } from '../NotesCard/NotesCard';
 import draftToHtml from 'draftjs-to-html';
 import htmlToDraft from 'html-to-draftjs';
 import 'react-draft-wysiwyg/dist/react-draft-wysiwyg.css';
 import './MyEditor.css';
 import 'draft-js/dist/Draft.css';
 
 const NotesPage = () => {
-	const initialValue = { title: '', body: '' };
-	const { notes, postNotes, getNotes } = useNotes();
-	const { sendNotesToArchive } = useArchive();
-	const [textField, setTextField] = useState(initialValue);
+	const { toggleModal, modal } = useModal();
+	const initialValue = { title: '', body: '', tag: 'None', priority: 'Low', date: '', cardColor: '#161b22' };
+	const { notes, postNotes } = useNotes();
+	const [formData, setFormData] = useState(initialValue);
 	const [editorState, setEditorState] = useState(EditorState.createEmpty());
 	const submitNote = () => {
 		postNotes({
-			title: textField.title,
-			body: textField.body,
-			tags: [],
+			title: formData.title,
+			body: formData.body,
+			tag: formData.tag,
+			priority: formData.priority,
+			cardColor: formData.cardColor,
 			createdAt: formatDate(),
 		});
 		setEditorState(EditorState.createEmpty());
-		setTextField(initialValue);
+		setFormData(initialValue);
+		toggleModal();
 	};
 
 	const titleChangeHandler = (e) => {
```

Comment:
Suggestion: Since you have a single object as the initial state for all the inputs, you could use a single function as a change handler by targetting event.target.name